.github/workflows/fix-review.yml

```
@@ -0,0 +1,80 @@
+name: Auto Fix Review
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)